app/models/comparison/annual_change_in_gas_out_of_hours_use.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: annual_change_in_gas_out_of_hours_uses
+#
+#  alert_generation_run_id                       :bigint(8)
+#  electricity_economic_tariff_changed_this_year :boolean
```

Comment:
I think the tariff changed column is different for gas?

Reply:
I think gas_economic_tariff_changed_this_year see from analytics:

```
    def self.tariff_changed_school_name(content_class = nil)
      if content_class.nil?
        { data: ->{ tariff_change_reference(addp_name, addp_etch || addp_gtch)}, name: :name, units: String, chart_data: true }
      else
        { data: ->{ tariff_change_reference(addp_name, addp_etch || addp_gtch)}, name: :name, units: String, chart_data: true, content_class: content_class }
      end
    end
```

Reply:
Yes, that's right its a different field.